packages/sfui/frameworks/react/components/SfInput/SfInput.tsx

```diff
@@ -20,7 +20,7 @@ const SfInput = polymorphicForwardRef<typeof defaultWrapperTag, SfInputProps>(
     return (
       <WrapperTag
         className={classNames([
-          'flex items-center gap-2 bg-white rounded-md ring-inset text-neutral-500 hover:ring-primary-700 focus-within:caret-primary-700 active:caret-primary-700 active:ring-primary-700 active:ring-2 focus-within:ring-primary-700 focus-within:ring-2',
+          'flex items-center gap-2 bg-white rounded-md after:ring-inset text-neutral-500 hover:ring-primary-700 focus-within:caret-primary-700 active:caret-primary-700 active:ring-primary-700 active:ring-2 focus-within:ring-primary-700 focus-within:ring-2',
```

Comment:
I'd have to check, but what if we just don't use `inset`, and standard ring instead? Would it resolve the issue?

Reply:
yes, also worked https://github.com/vuestorefront/storefront-ui/pull/2714/commits/8dd325e2088a78e8adf30fa26355c741fb7ff88e